packages/apps/tools/src/components/Common/DrawerToolbar/styles.css.ts

```diff
@@ -0,0 +1,67 @@
+import { sprinkles, vars } from '@kadena/react-ui/theme';
+
+import { style } from '@vanilla-extract/css';
+
+export const gridItemCollapsedSidebarStyle = style([
+  sprinkles({
+    position: 'fixed',
+    right: 0,
+    top: '$16',
+    bottom: '$10',
+    backgroundColor: '$neutral1',
```

Comment:
use of neutral colour, good idea?

Reply:
on light mode it is set to #fafafa which is neutral1, and on dark mode closest color to #111111 is also neutral1